### PR TITLE
✨  Create implementation of `<bento-reddit>`

### PR DESCRIPTION
**Pull request by rbeckthomas**
This PR creates the initial implementation for the `bento-reddit` component.

### CODE REVIEW

File: src/bento/components/bento-reddit/1.0/bento-reddit.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * - size-defined element
+ */
+ bento-reddit{
+  display: block;
+  overflow: hidden;
+  position: relative;
+}
+
+/* Pre-upgrade: size-defining element - hide text. */
+amp-twitter:not(.i-amphtml-built) {
```
Comment (dethstrobe) [inline src/bento/components/bento-reddit/1.0/bento-reddit.css:13]: Just to clarify, but is this suppose to be `amp-twitter`?

File: src/bento/components/bento-reddit/1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+  type="text/css"
+  href="https://cdn.ampproject.org/v0/bento-reddit-1.0.css"
+/>
+```
+
+Alternatively, you may also make the light-weight pre-upgrade styles available inline:
+
+```html
+<style>
+  bento-facebook {
```
Comment (alanorozco) [inline src/bento/components/bento-reddit/1.0/README.md:114]:
```suggestion
  bento-reddit {
```

**Review by alanorozco — APPROVED** [review]
LGTM!

File: src/bento/components/bento-reddit/OWNERS [hunk 3]
```diff
@@ -0,0 +1,10 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/main/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [{name: 'ampproject/wg-components'}, {name: 'pablos1'}],
```
Comment (kvchari) [inline src/bento/components/bento-reddit/OWNERS:7]: who is pablos1? and can we just remove this and create an owners file for src/bento/components that lists wg-components as the owner?

Comment (rbeckthomas) [reply src/bento/components/bento-reddit/OWNERS:7]: That would be a copy pasta error from the bento-jwplayer owners file. Right now each amp-component has its own OWNERS, Im not sure if there is a structural reason to keep it that way though.
@alanorozco What do you think?

**Pull request merged**